test/cpp/api/functional.cpp

```diff
@@ -687,6 +687,15 @@ TEST_F(FunctionalTest, PReLU) {
   ASSERT_TRUE(torch::allclose(y, y_exp));
 }
 
+TEST_F(FunctionalTest, LayerNorm) {
+  const auto input = torch::randn({2, 2});
+  const auto weight = torch::randn({2, 2});
+  const auto bias = torch::randn({2, 2});
+  auto y = F::layer_norm(input, LayerNormOptions({2, 2}).eps(2e-5), weight, bias);
```

Comment:
In order to test the "no weights and no bias" case for `F::layer_norm`, let's probably change this to
```suggestion
  auto y = F::layer_norm(input, LayerNormOptions({2, 2}).eps(2e-5));
```
and remove `weight` and `bias` for this test.